docs/book/src/tasks/experimental-features/runtime-sdk/implement-topology-mutation-hook.md

```diff
@@ -95,15 +95,17 @@ status:
                 example: "internal.com"
                 description: "comma-separated list of machine or domain names excluded from using the proxy."
       - name: http-proxy
+        # definitionsConflict is true if there are non-equal definitions for a variable with the same name.
```

Comment:
```suggestion
        # definitionsConflict is true if there are non-equal definitions for a variable.
```
nit: up to you
In my opinion slightly simpler to read and "for a variable" already implies that it's for the same variable/name.